src/Agent.Worker/ExecutionContext.cs

```diff
@@ -689,13 +692,45 @@ public string TranslatePathForStepTarget(string val)
 
         public ContainerInfo StepTarget()
```

Comment:
It might be nicer to always return a non-null value from this method. Unless you need to return null in some cases.

Reply:
That would definitely be a better design. I'll see what I can do.